src/main/java/org/jenkinsci/test/acceptance/po/Control.java

```diff
@@ -186,15 +195,22 @@ public void set(Object text) {
      */
     public void selectDropdownMenu(Class type) {
         click();
-        findCaption(type,findDropDownMenuItem).click();
-        elasticSleep(1000);
+        WebElement we = findCaption(type,findDropDownMenuItem);
+        // the element may not yet be visible so wait for it to become shown after the click above
+        waitFor(we).pollingEvery(100L, TimeUnit.MILLISECONDS).withTimeout(time.milliseconds(1000), TimeUnit.MILLISECONDS).until(() -> we.isDisplayed());
```

Comment:
I see now that `time` is an ElasticTime .... can ElasticTime return a Duration maybe ?

Reply:
we can not use Duration in this `waitFor`  the duration is defined in the `FluentWait` super class and then we can not use the `until` methods defined in our `Wait`